### PR TITLE
Support rules_oci + bzlmod

### PR DESCRIPTION
**Pull request by opicaud**
We can remove or keep the two first changes now or later, as they are "convenience" commits
This pull request concers:
- 1 fix about the replacement of templated value in the values.yaml
- https://github.com/abrisco/rules_helm/issues/12
- https://github.com/abrisco/rules_helm/issues/15

`image` rule must have been replaced by `oci_images` from commit b7fb0af as `rules_docker` has been removed to support bzlmod

### CODE REVIEW

File: helm/helm.bzl [hunk 1]
```diff
@@ -3,7 +3,7 @@
 load("//helm/private:helm_install.bzl", "helm_install", "helm_push", "helm_reinstall", "helm_uninstall")
 load("//helm/private:helm_package.bzl", "helm_package")
 
-def helm_chart(name, images = [], deps = None, tags = [], install_name = None, **kwargs):
+def helm_chart(name, oci_images = [], deps = None, tags = [], install_name = None, **kwargs):
```
Comment (abrisco) [inline helm/helm.bzl:6]: I don't think you need to change the parameter name here, can you leave the interface the same so others don't need to make as many changes when udpating?

**Review by abrisco — CHANGES_REQUESTED** [review]
Hi @opicaud, I'm so so sorry for the delay here. Some stuff came up in my personal life that was extremely time consuming. I think this change is fantastic! I had just one request but otherwise, after reading through the rules_oci issues you engaged in, I think this work is phenomenal. Thank you so much!

File: helm/helm.bzl (hunk 1 above)
Comment (abrisco) [reply helm/helm.bzl:6]: this is actually something I can do post-merge

**Review by abrisco — APPROVED** [review]

**Comment by opicaud** [comment]
Hey @abrisco 
Glad you are back :) and thanks to have reviewed and accepted this pull request.
To terminate this feature, I propose to:
- update the setup of the README with the explanation about how to install rules_helm via a MODULE.bazel
- publish rules_helm as component of the Bazel Central Registry --> see https://github.com/bazelbuild/bazel-central-registry

**Comment by abrisco** [comment]
> Hey @abrisco 
> 
> Glad you are back :) and thanks to have reviewed and accepted this pull request.
> 
> To terminate this feature, I propose to:
> 
> - update the setup of the README with the explanation about how to install rules_helm via a MODULE.bazel
> 
> - publish rules_helm as component of the Bazel Central Registry --> see https://github.com/bazelbuild/bazel-central-registry

Would you be willing to update the README? I can look into updating the release pipeline to publish there.

**Pull request merged**